docs/website/docs/examples/incremental_loading/code/run-snippets.py

```diff
@@ -0,0 +1,41 @@
+
+
+def incremental_snippet() -> None:
```

Comment:
that was ok for snippets but is not OK for examples. examples are intended to be visible in full and be runnable with python name.py.
this is IMO convenient for us but not for the user. what I'd do is an example structure that look like this:
```
docs/examples
  |--incremental_loading
         |-.dlt/
         |-incremental_loading.py
         |-README.md
```
where README.md is your index.md (I hope that's possible). then we give people actual github experience

Reply:
Well I have built a script which exports this examples as runnable projects to /docs/examples and removes all our snipync directives. So we can have nice tests for us and the user can just run it. You can see this in this PR. All the stuff in docs/examples in generated from the stuff in docs/website/docs/examples

Reply:
heh that's even better. didn't see that. sorry for stupid comment :)